### PR TITLE
Add rubygem-robotex from katello-packaging

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: katello/rubygem-robotex/rubygem-robotex.spec [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+%define rubyabi 1.9.1
+
+Summary:    Ruby library to obey robots.txt 
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    1.0.0
+Release:    19%{?dist}
+License:    MIT 
+Group:      Development/Languages
+URL:        http://www.github.com/chriskite/robotex
```
Comment (ekohl) [inline katello/rubygem-robotex/rubygem-robotex.spec]: https

File: katello/rubygem-robotex/rubygem-robotex.spec [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+%{?scl:"}
+%endif
+popd
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/lib
+%exclude %{gem_cache}
+%{gem_spec}
+%doc %{gem_instdir}/LICENSE
```
Comment (ekohl) [inline katello/rubygem-robotex/rubygem-robotex.spec]: %license?

**Comment by ehelms** [comment]
Note I put this under `katello/` to differentiate these. I have a thread on `-dev` around doing this for all packages to better categorize them do to sheer numbers.

**Comment by ehelms** [comment]
Needs -- https://github.com/theforeman/foreman-infra/pull/373

**Comment by ehelms** [comment]
You can see it passed here because I made sure to test the infra change.

**Comment by ehelms** [comment]
Removal PR -- https://github.com/Katello/katello-packaging/pull/572

**Pull request merged**